packages/d3fc-series/src/webgl/ohlc.js

```diff
@@ -0,0 +1,74 @@
+import ohlcBase from '../ohlcBase';
+import functor from '../functor';
+import { glOhlc, scaleMapper } from '@d3fc/d3fc-webgl';
+import { rebindAll, exclude, rebind } from '@d3fc/d3fc-rebind';
+
+export default () => {
+    const base = ohlcBase();
+    let lineWidth = functor(1.0);
+
+    let draw = glOhlc();
+
+    const ohlc = (data) => {
+        const filteredData = data.filter(base.defined());
+
+        const xScale = scaleMapper(base.xScale());
+        const yScale = scaleMapper(base.yScale());
+
+        const accessor = getAccessors();
+
+        const xValues = new Float32Array(filteredData.length);
+        const open = new Float32Array(filteredData.length);
+        const high = new Float32Array(filteredData.length);
+        const low = new Float32Array(filteredData.length);
+        const close = new Float32Array(filteredData.length);
+        const bandwidths = new Float32Array(filteredData.length);
+        const lineWidths = new Float32Array(filteredData.length);
+
+        filteredData.forEach((d, i) => {
+            xValues[i] = xScale.scale(accessor.xValues(d, i));
+            open[i] = yScale.scale(accessor.open(d, i));
+            high[i] = yScale.scale(accessor.high(d, i));
+            low[i] = yScale.scale(accessor.low(d, i));
+            close[i] = yScale.scale(accessor.close(d, i));
+            bandwidths[i] = accessor.bandwidth(d, i);
+            lineWidths[i] = lineWidth(d, i);
```

Comment:
Looks like this eventually gets loaded in as an attribute. Is there a reason it's not done as a uniform? Then you wouldn't have to repeat all the values for each vertex

Reply:
This was so that each data point could have a different line width, although looking now I don't think this is supported for svg or canvas. I can change this to be a uniform, similar to how `glLine` implements `width`, if we don't want this behaviour just for the webgl implementation?

Reply:
Yea please swap it out for a uniform. I can't think of a good reason for it.